src/vm/core.zig

```diff
@@ -913,3 +941,60 @@ test "update fp dst felt" {
         42,
     );
 }
+
+test "trace is enabled" {
+    // ************************************************************
+    // *                 SETUP TEST CONTEXT                       *
+    // ************************************************************
+    // Initialize an allocator.
+    var allocator = std.testing.allocator;
+
+    // Create a new VM instance.
+    var config = Config{ .proof_mode = false, .enable_trace = false };
+    config.enable_trace = true;
```

Comment:
why dont you put directly the true value when initiating ?

Reply:
oh initially i was using `Config.default()` there, then used regex to replace it :). will update.